modules/040-control-plane-manager/images/kube-apiserver/werf.inc.yaml

```diff
@@ -14,12 +14,19 @@ import:
   add: /src/_output/bin/kube-apiserver
   to: /usr/bin/kube-apiserver
   before: setup
+- artifact: {{ $.ModuleName }}/{{ $.ImageName }}-nsswitch
+  add: /etc/nsswitch.conf
+  to: /etc/nsswitch.conf
+  before: setup
+docker:
+  ENTRYPOINT: ["/usr/bin/kube-apiserver"]
+{{- end }}
+---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-nsswitch
+from: {{ $.Images.BASE_ALPINE}}
 shell:
   beforeInstall:
   # set up nsswitch.conf for Go's "netgo" implementation
   # Go stdlib completely ignores /etc/hosts file without it
-  # https://github.com/moby/moby/issues/34544
+  # https://github.com/golang/go/issues/22846
   - "echo 'hosts: files dns' > /etc/nsswitch.conf"
```

Comment:
Should we add this to base distroless image ?

Reply:
It's already there. So, perhaps, it would be better to just completely remove it from control-plane-manager?
![image](https://github.com/deckhouse/deckhouse/assets/950313/d766c9fe-cc46-44eb-844a-36a95edc2e99)